test-clients/src/main/java/com/hedera/services/bdd/spec/infrastructure/meta/ContractResources.java

```diff
@@ -1083,23 +1082,23 @@ public class ContractResources {
 			"\"nonpayable\", \"type\": \"function\" }";
 
-	public static final String CREATE_FUNGIBLE_SCENARIO_5 = "{\"inputs\":[{\"internalType\":\"bytes\",\"name\":\"ed25519AdminKey\",\"type\":\"bytes\"},{\"internalType\":\"address\",\"name\":\"autoRenewAccount\",\"type\":\"address\"},{\"internalType\":\"uint32\",\"name\":\"autoRenewPeriod\",\"type\":\"uint32\"}],\"name\":\"createTokenThenQueryAndTransfer\",\"outputs\":[{\"internalType\":\"address\",\"name\":\"createdTokenAddress\",\"type\":\"address\"}],\"stateMutability\":\"payable\",\"type\":\"function\"}";
-
-	public static final String CREATE_FUNGIBLE_ABI = "{ \"inputs\": [ { \"internalType\": \"address\", " +
-			"\"name\": \"contractKey\", \"type\": \"address\" } ], \"name\": \"createFungible\", " +
-			"\"outputs\": [ { \"internalType\": \"address\", \"name\": \"createdTokenAddress\", " +
-			"\"type\": \"address\" }], \"stateMutability\": \"payable\", \"type\": \"function\" }";
-
-	public static final String CREATE_FUNGIBLE_WITH_FEES_ABI = "{\"inputs\":[{\"internalType\":\"bytes\",\"name\":\"ed25519Key\",\"type\":\"bytes\"},{\"internalType\":\"bytes\",\"name\":\"ecdsaKey\",\"type\":\"bytes\"}],\"name\":\"createFungibleWithFees\",\"outputs\":[{\"internalType\":\"address\",\"name\":\"createdTokenAddress\",\"type\":\"address\"}],\"stateMutability\":\"payable\",\"type\":\"function\"}";
-
-	public static final String CREATE_NON_FUNGIBLE_ABI = "{ \"inputs\": [], \"name\": \"createNonFungibleToken\", " +
-			"\"outputs\": [ { \"internalType\": \"address\", \"name\": \"createdTokenAddress\", " +
-			"\"type\": \"address\" } ], \"stateMutability\": \"payable\", \"type\": \"function\" }";
-
-	public static final String CREATE_NON_FUNGIBLE_WITH_FEES_ABI = "{ \"inputs\": [ { \"internalType\": \"address\", " +
-			"\"name\": \"feeCollector\", \"type\": \"address\" } ], \"name\": \"createNonFungibleTokenWithCustomFees\", " +
-			"\"outputs\": [ { \"internalType\": \"address\", \"name\": \"createdTokenAddress\", " +
-			"\"type\": \"address\" } ], \"stateMutability\": \"payable\", \"type\": \"function\" }";
+	public static final String TOKEN_CREATE_FUNGIBLE_WITHOUT_FEES_ABI = "{\"inputs\":[{\"internalType\":\"address\",\"name\":\"treasury\",\"type\":\"address\"},{\"internalType\":\"bytes\",\"name\":\"ed25519\",\"type\":\"bytes\"},{\"internalType\":\"bytes\",\"name\":\"ecdsa\",\"type\":\"bytes\"},{\"internalType\":\"address\",\"name\":\"contractID\",\"type\":\"address\"},{\"internalType\":\"address\",\"name\":\"delegatableContractID\",\"type\":\"address\"},{\"internalType\":\"address\",\"name\":\"autoRenewAccount\",\"type\":\"address\"},{\"internalType\":\"uint32\",\"name\":\"autoRenewPeriod\",\"type\":\"uint32\"},{\"internalType\":\"address\",\"name\":\"toAssociateWith\",\"type\":\"address\"}],\"name\":\"createTokenWithKeysAndExpiry\",\"outputs\":[{\"internalType\":\"address\",\"name\":\"createdTokenAddress\",\"type\":\"address\"}],\"stateMutability\":\"payable\",\"type\":\"function\"}";
```

Comment:
This would be more readable if you used """ blocks:

```
public static final String TOKEN_CREATE_FUNGIBLE_WITHOUT_FEES_ABI = """
{ "inputs":  ...
""";

Reply:
Even better is to make use of new lines and then remove them.. i.e.:
```
public string final String TOKEN_CREATE_FUNGIBLE_WITHOUT_FEES_ABI = """
{  
  "inputs":  [{"internalType":"bytes",
                      "name":   "ecdsaAdminKey",
                      "type":     "bytes"}]
}
""".replace("\n", "").replace(" ", "");
```

Reply:
There is actually a [PR](https://github.com/hashgraph/hedera-services/pull/2913) to clean all this up, it has just temporarily slipped a bit. 😞